src/pytorch_lightning/profiler/__init__.py

```diff
@@ -11,12 +11,70 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from pytorch_lightning.profilers.advanced import AdvancedProfiler
-from pytorch_lightning.profilers.base import PassThroughProfiler
-from pytorch_lightning.profilers.profiler import Profiler
-from pytorch_lightning.profilers.pytorch import PyTorchProfiler
-from pytorch_lightning.profilers.simple import SimpleProfiler
-from pytorch_lightning.profilers.xla import XLAProfiler
```

Comment:
https://github.com/Lightning-AI/lightning/pull/12308 made a mistake, importing from `profilerS` instead of `profiler`.

Meaning that users that were importing from `pytorch_lightning.profiler` did not get the deprecation.

So I added it again below